tmp/build/patch-prometheus.sh

```diff
@@ -23,12 +23,15 @@ function prometheus_patch_deployment() {
           volumeMounts:\
           - mountPath: /etc/tls/private\
             name: secret-prometheus-tls\
+          - mountPath: /etc/proxy/htpasswd\
+            name: secret-prometheus-htpasswd\
           args:\
           - -provider=openshift\
           - -https-address=:3001\
           - -http-address=\
           - -email-domain=*\
           - -upstream=http://localhost:9090\
+          - -htpasswd-file=/etc/proxy/htpasswd/auth\
```

Comment:
Should we also add `-display-htpasswd-form=false`?  If I understand correctly, this would prevent external users from being able to use the generated password to access prometheus.

Reply:
Done